.misc/appveyor.yml

```diff
@@ -24,6 +24,10 @@ branches:
     - /^sils\/.*/
 
 install:
+  # curl exists in this path. See this discussion for more details:
+  # http://help.appveyor.com/discussions/problems/6312-curl-command-not-found
+  - "SET PATH=C:\\Program Files\\Git\\mingw64\\bin;%PATH%"
```

Comment:
The new addition should be after the existing PATH, not before.
And it appears we don't need those quotes around the command.
If we remove them, we dont need the backslash escapes which appear in GitHub as syntax errors.
c.f. https://github.com/jayvdb/coala/commit/c5aadba1f76b5963e5b759f1ba403eb2cfbfd45b and https://ci.appveyor.com/project/jayvdb17618/coala/build/1.0.15